packages/drivers/driver-web-cache/src/FluidCacheIndexedDb.ts

```diff
@@ -13,14 +13,15 @@ import { FluidCacheErrorEvent } from "./fluidCacheTelemetry";
 export const FluidDriverCacheDBName = "fluidDriverCache";
 
 // The name of the object store within the indexed db instance that the driver will use to cache Fluid content.
-export const FluidDriverObjectStoreName = "driverStorage.V3";
+export const FluidDriverObjectStoreName = "driverStorage.V4";
 
-export const CurrentCacheVersion = 3;
+export const CurrentCacheVersion = 4;
 
 // Note that V1 and V2 were misspelled as "diver", and we need to keep using the misspelling here.
 export const oldVersionNameMapping: Partial<{ [key: number]: string }> = {
```

Comment:
I see a fundamental issue with this versioning schema, which we should address (in a separate PR / task - please open a new task). This code assumes that there is only one app in town. Or multiple but running same code (on same version of cache). This is not true, Teams, OWA, Loop may be on different versions of driver, opening same files. More recent app (likely Loop) will keep deleting caches created by other apps, and other apps would do what? Downgrade schema? Not sure.

I think cache should be either partitioned by app (helps with 95% of that problem, does not help with code rollbacks and sharing of data), or we should find other strategy on how to deal with versioning and sharing.

This seems like not well thought out.

Reply:
What scenario has more than one app running on the same domain? This is Indexeddb's standard versioning mechanism and is isolated to the domain it was created on

Reply:
Ah, good point, for some reason I thought it's in shared space